### ISSUE TITLE
WCAG 4.1.1,WCAG 4.1.2: ARIA attributes must conform to valid values (#TextField1550)

### ISSUE THREAD

**Issue by joshmccall221** [OfficeDev/office-ui-fabric-react#10411]
![image](https://user-images.githubusercontent.com/7776611/64652693-52adda00-d3d9-11e9-9ae2-f3b6d77c9b39.png)

#### Issue
When label is provided, aria is not correctly set.

`ARIA attributes must conform to valid values` ([aria-valid-attr-value](https://dequeuniversity.com/rules/axe/3.2/aria-valid-attr-value?application=msftAI))
#### Target application
[Home - Office UI Fabric](https://developer.microsoft.com/en-us/fabric#/controls/web/textfield)
#### Element path
#TextField1550
#### Snippet
`<input type="text" id="TextField1550" aria-labelledby="TextFieldLabel1552" class="ms-TextField-field field-623" aria-describedby="TextFieldDescription1551" aria-invalid="false" value="">`
#### How to fix
    Fix all of the following:
      Invalid ARIA attribute value: aria-labelledby="TextFieldLabel1552"
#### Environment
Chrome version 76.0.3809.132
This accessibility issue was found using Accessibility Insights for Web 2.9.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by natalieethell** [comment]
Hey @joshmccall221 thanks for submitting an issue. It looks like this is caused by a check in TextField that sets `aria-labelledby` based on the existence of `props.label`. Since this example uses its own custom rendered label, it actually doesn't need to pass in `label` to TextField. Should be an easy fix, I'll open a PR soon.

**Comment by natalieethell** [comment]
Actually, a better fix might be to just conditionally set the `aria-labelledby` based on if `onRenderLabel` is passed in. I'll look at both solutions.
